spec/controllers/members_controller_spec.rb

```diff
@@ -164,7 +164,7 @@
     end
 
     context "with yet-to-be-invited emails" do
-      let(:emails) { ["h.wurst@openproject.com", "l.lustig@openproject.com"] }
+      let(:emails) { ["h.wurst@openproject.com", "12775518238@qq.com"] }
```

Comment:
Thanks @Eric-Guo , the test looks good 👍, however I would prefer either using the `@openproject.com` domain in the test or a domain that is reserved for testing such as `@example.com`.